### DISCUSSION TITLE
Prune this GIT repo?

### DISCUSSION THREAD

**Discussion by imagingbook** [Digital-Media/HagenbergThesis#118] [Ideas]
Due to the large number of binary files, this repo has grown a lot over time. A full clone now requires around 650MB disk space, while the actual data is only about 33MB. So there is a LOT of unnecessary stuff hanging around.
Should we backup and prune this repository and only keep a short history?

**Comment by hochleitner** [comment]
I have no issues with that. I've been struggling a long time wether we should keep the PDFs in the repo at all but for preview's sake I guess it is necessary. Same with the zipped documents.

I have used the [BFG Repo-Cleaner](https://rtyley.github.io/bfg-repo-cleaner/) in the past to get rid of big files, it's an easy way to remove unwanted files.

**Comment by imagingbook** [comment]
Done - although some changes could not be pushed ("deny updating a hidden ref"). But the repo's size is now down to 66MB ...

**Reply by imagingbook** [reply]
You should do a fresh clone to avoid that old stuff is pushed back again.

**Reply by hochleitner** [reply]
Thanks, I did that. And since a fresh clone is needed anyway, I renamed the `master` branch to `main` to keep up with the recent discussions (see https://github.com/github/renaming).

**Reply by imagingbook** [reply]
OK, whatever difference it makes ...
